src/test/scala/com/databricks/labs/mosaic/core/geometry/Test_MosaicGeometryJTS.scala

```diff
@@ -0,0 +1,20 @@
+package com.databricks.labs.mosaic.core.geometry
+
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers.convertToAnyShouldWrapper
+
+class Test_MosaicGeometryJTS extends AnyFunSuite {
```

Comment:
shouldn't this also test for a true result?

Reply:
I was just noticing that there is only a false return test, was wondering if there should also be a separate test that shows a true return?